apps/demo-app/src/views/HomeView.vue

```diff
@@ -79,6 +79,7 @@ const show = computed(() => {
 const densityOptions = DENSITIES.map((value: string) => ({
   value,
   label: capitalize(value),
+  skeleton: false,
```

Comment:
I don't think this does anything..?
```suggestion
```

Reply:
I've added this line because when we switch to skeleton mode, the density radio group is  picking up on the provided property and sets the density options to skeleton too. I don't think this is out intention.